src/validators/duration_distance.rs

```diff
@@ -126,7 +126,15 @@ fn validate_speeds(gtfs: &gtfs_structures::Gtfs) -> Result<Vec<Issue>, gtfs_stru
                             .add_related_object(&*arrival.stop)
                             .details(&details)
                     });
-                    issue.push_related_object(trip);
```

Comment:
No tests broke with this change, which shows it would be a good idea to add a test.

Reply:
I implemented a proper test with a dedicated fixture.